charts/trino/values.yaml

```diff
@@ -6,7 +6,7 @@ image:
   repository: trinodb/trino
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart version.
-  tag: latest
+  tag: ~
```

Comment:
this will make CI not useful since we'll never know of breaking changes coming from Trino.

Also since the Trino release is not synced up with chart releases this means that the chart will always be pointing at old releases when we want people to be running latest versions.

For people who want to pin versions they'll surely know how to provide required values.

Reply:
I'm not sure I understand why it will make CI not useful.
CI script will run tests with `.Chart.AppVersion` value, so we can guaranty that chart works with specific version of Trino. As I know tag `latest` is a bad practice because it might represent different actual versions for yesterday, today and tomorrow. It means that the result of the running **CI** or **helm install** is not repeatable. I case of breaking changes in Trino the chart should be explicitly fixed/updated. Also we can update Trino version in the chart periodically.

Reply:
Also I think it's confusing that the following shows 372 as the current version of Trino while actual latest version is 374
```
> helm search repo
NAME                    CHART VERSION   APP VERSION     DESCRIPTION
trino/trino             0.5.0           372             Fast distributed SQL query engine for big data ...
```

Reply:
We should first add a mechanism that runs CI against latest version (as well as app version) before this PR then.

Reply:
This change actually says we will get default version from `.Chart.AppVersion`
We can change `appVersion` in `chart.yaml` to `latest` in case we want point chart to latest version of Trino

Reply:
Also I'm pretty sure that tying Chart versions and Trino versions is not bad. That's how we can say and guaranty that specific version of chart works at least with specific version of Trino. Otherwise it might be a case when I run `helm install` and it doesn't work because yesterday incompatible version of Trino has been released.

Reply:
Tying them is not bad but right now there is no way to co-ordinate releases across Trino and charts. It's something that would eventually be good but not feasible to do today.

Reply:
Can I resolve this conversation?